### PR TITLE
incoming liquidity

### PR DESCRIPTION
**Pull request by rsbondi**
per #4 

also minor styling adjutstment

### CODE REVIEW

File: forms/ui_channelsPage.py [hunk 1]
```diff
@@ -208,16 +208,27 @@ def setupUi(self, ChannelsPage):
...
-        self.line_3 = QtWidgets.QFrame(self.verticalFrame)
-        self.line_3.setFrameShape(QtWidgets.QFrame.HLine)
-        self.line_3.setFrameShadow(QtWidgets.QFrame.Sunken)
-        self.line_3.setObjectName("line_3")
-        self.layoutAmount.addWidget(self.line_3)
+        self.line_ = QtWidgets.QFrame(self.verticalFrame)
+        self.line_.setFrameShape(QtWidgets.QFrame.HLine)
+        self.line_.setFrameShadow(QtWidgets.QFrame.Sunken)
+        self.line_.setObjectName("line_")
+        self.layoutAmount.addWidget(self.line_)
```
Comment (darosior) [inline forms/ui_channelsPage.py]: This is not coherent with the xml file (`s/line_/line_3`), did you modify this file by hand ?

**Review by darosior — COMMENTED** [review]
Tested, work well. Just something weird with the generated ui file

File: forms/ui_channelsPage.py (hunk 1 above)
Comment (rsbondi) [reply forms/ui_channelsPage.py]: apologies, I have re-generated this file and should now be in sync

**Comment by darosior** [comment]
Thanks !

**Pull request merged**